.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
```

Comment:
i suggest a regex for numbers, e.g. `tags: [ 'v[0-9]+\.[0-9]+\.[0-9]+' ]`

Reply:
OK, but we can add that constraint at the step that pushes the container to GHCR. I.e., we can run the workflow "liberally" but restrict when we push the (tagged) image.

Reply:
hmm, what is the use case for pushing a tag that's not formatted in `'v[0-9]+\.[0-9]+\.[0-9]+'`?